.test-infra/jenkins/common_job_properties.groovy

```diff
@@ -245,7 +245,7 @@ class common_job_properties {
 
     context.publishers {
       // Notify an email address for each failed build (defaults to commits@).
-      mailer(notifyAddress, false, emailIndividuals)
+      mailer(notifyAddress, false, false)
```

Comment:
I'd say either leave the variable or add a comment that this is explicitly turned off. Whatever works.

Reply:
Commented like whoa. Remove that variable and another that we never change.